samples/csharp_dotnetcore/14.nlp-with-dispatch/appsettings.json

```diff
@@ -1,4 +1,33 @@
 {
-    "botFilePath": "nlp-with-dispatch.bot",
-    "botFileSecret": ""
-}
\ No newline at end of file
+  "Logging": {
```

Comment:
This is an interesting discussion - because KEY's are going to be put in here, we don't want developers to check this file into their source control system (Github, etc). Yet, for the sample, we do want it. 

We need to figure out our story around the .Net Core 2.1 Secret Management soon.